src/routes/patches/PatchItem.svelte

```diff
@@ -117,6 +122,10 @@
 		color: var(--accent-color-two);
 	}
 
+	.button{
+		display: flex;
+		align-items: center;
+	}
```

Comment:
Is this class necessary?

Reply:
`display: contents` would work here except according to https://adrianroselli.com/2022/07/its-mid-2022-and-browsers-mostly-safari-still-break-accessibility-via-display-properties.html that would change the semantics.

Reply:
> Is this class necessary?

This class removes the list style and centers the arrow button. Without it, the arrow button will have a bullet point behind it and will be slightly off-center vertically.